interface/patient_file/ccr_import.php

```diff
@@ -22,39 +22,34 @@
  * @author  Ajil P M <ajilpm@zhservices.com>
  * @link    http://www.open-emr.org
  */
+require_once(dirname(__FILE__) . "../../globals.php");
 
-
-
-require_once(dirname(__FILE__) . "/../globals.php");
+use OpenEMR\Core\Header;
 
 ?>
 <html>
 <head>
+<?php Header::setupHeader(['opener']); ?>
```

Comment:
change this to:
``````
<?php Header::setupHeader(); ?>
``````
This is a really cool function that @robertdown brought in to standardize use of assets. For example, the above line brings in the following code:
``````
<link rel="stylesheet" href="/openemr/interface/themes/style_light.css?v=30" type="text/css">
<link rel="stylesheet" href="/openemr/public/assets/bootstrap-3-3-4/dist/css/bootstrap.min.css?v=30" type="text/css">
<link rel="stylesheet" href="/openemr/public/assets/font-awesome-4-6-3/css/font-awesome.min.css?v=30" type="text/css">

<script type="text/javascript" src="/openemr/public/assets/jquery-min-3-1-1/index.js?v=30"></script>
<script type="text/javascript" src="/openemr/public/assets/bootstrap-3-3-4/dist/js/bootstrap.min.js?v=30"></script>
<script type="text/javascript" src="/openemr/library/textformat.js?v=30"></script>
<script type="text/javascript" src="/openemr/library/dialog.js?v=30"></script>
``````